src/iOS.Core/Controllers/LoginAddViewController.cs

```diff
@@ -75,17 +85,35 @@ public override void ViewDidLoad()
 
             PasswordCell.TextField.SecureTextEntry = true;
             PasswordCell.TextField.ReturnKeyType = UIReturnKeyType.Next;
+            PasswordCell.Button.TitleLabel.Font = UIFont.FromName("bwi-font", 28f);
+            PasswordCell.Button.SetTitle(BitwardenIcons.Generate, UIControlState.Normal);
+            PasswordCell.Button.TouchUpInside += (sender, e) =>
+            {
+                PerformSegue("passwordGeneratorSegue", this);
+            };
+
+            PasswordCell.SecondButton.TitleLabel.Font = UIFont.FromName("bwi-font", 28f);
+            PasswordCell.SecondButton.SetTitle(BitwardenIcons.Eye, UIControlState.Normal);
+            PasswordCell.SecondButton.TouchUpInside += (sender, e) =>
+            {
+                if (PasswordCell.TextField.SecureTextEntry)
+                {
+                    PasswordCell.SecondButton.SetTitle(BitwardenIcons.EyeSlash, UIControlState.Normal);
+                    PasswordCell.TextField.SecureTextEntry = false;
+                }
+                else
+                {
+                    PasswordCell.SecondButton.SetTitle(BitwardenIcons.Eye, UIControlState.Normal);
+                    PasswordCell.TextField.SecureTextEntry = true;
+                }
+            };
```

Comment:
This is repeated in several places like `BaseLockPasswordViewController`, `LockPasswordViewController`. Made a class that can configure the cell and button to have password behavior or add a Builder to create cells with different behaviors/uses